prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirToHl7ConverterTests.kt

```diff
@@ -445,8 +446,12 @@ class FhirToHl7ConverterTests {
 
         // Use a file based schema which will fail as we do not have enough data in the bundle
         val transformer = FhirToHl7Converter(
-            "classpath:/fhirengine/translation/hl7/schema/schema-read-test-01/ORU_R01.yml",
-            mockk<BlobAccess.BlobContainerMetadata>(), warnings = mutableListOf(), errors = mutableListOf()
+            SchemaReferenceResolverHelper.retrieveHl7SchemaReference(
+                "classpath:/fhirengine/translation/hl7/schema/schema-read-test-01/ORU_R01.yml",
+                mockk<BlobAccess.BlobContainerMetadata>()
+            ),
+            warnings = mutableListOf(),
+            errors = mutableListOf()
```

Comment:
These tests should be refactored to use file or classpath schemas instead. It can happen now or in #18423 (add a note there if so).